src/jtag/drivers/oscan1_ftdi.c

```diff
@@ -0,0 +1,1538 @@
+/**************************************************************************
+*   Copyright (C) 2012 by Andreas Fritiofson                              *
```

Comment:
I'm not sure what the copyright should be, but this doesn't look right. In the RISC-V specific code I've not bothered with a copyright line at all, nor with the GPL info. Upstream has accepted that without comment.